src/components/AgentCardsContainer/index.jsx

```diff
@@ -0,0 +1,49 @@
+import AgentCard from 'components/shared/AgentCard';
+import PropTypes from 'prop-types';
+import React, { useEffect, useState } from 'react';
+import { getAgents } from 'services/fake-agents-service';
+import { CardsContainer, Container } from './agent-cards-container.styles';
+
+/**
+ * Component holding the cards of agents, it maps agents to their cards.
+ *
+ * @return {JSX.Element} Container holding agent cards.
+ */
+function AgentCardsContainer({ filteredTarget }) {
+  const [agents, setAgents] = useState([]);
+
```

Comment:
Please add documentation for any function
```suggestion

  // fetch agent data
```